keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -630,22 +630,21 @@ The metric will use the following labels:
 
 The following audit annotations will be added:
```

Comment:
made coherent with the current implementation and the audit tracking issue in https://github.com/kubernetes/kubernetes/issues/103923